theano/sandbox/gpuarray/dnn.py

```diff
@@ -1783,6 +1395,22 @@ def local_softmax_dnn(node):
         return [out]
```

Comment:
Should that one check for the version, because of the bug when taking softmax on (1, 1) images?